playbooks/common/openshift-cluster/upgrades/upgrade_nodes.yml

```diff
@@ -23,35 +29,59 @@
     retries: 10
     delay: 5
     register: node_unschedulable
-    until: node_unschedulable|succeeded
+    until: node_unschedulable is succeeded
 
   - name: Drain Node for Kubelet upgrade
     command: >
-      {{ hostvars[groups.oo_first_master.0].openshift.common.admin_binary }} drain {{ openshift.node.nodename | lower }} --config={{ openshift.common.config_base }}/master/admin.kubeconfig --force --delete-local-data --ignore-daemonsets
+      {{ hostvars[groups.oo_first_master.0]['first_master_client_binary'] }} adm drain {{ openshift.node.nodename | lower }}
```

Comment:
This change doesn't seem to be related to node hooks, is it required?

Reply:
Feat/nodeupgradehooks 3.9 was good ... will reverify the code and fix it :-)